mongo/test/org/immutables/mongo/bson4gson/JavaTimeTest.java

```diff
@@ -93,7 +94,7 @@ public void localDateTime() throws IOException {
 
   @Test
   public void instant() throws IOException {
-    Instant now = Instant.now();
+    Instant now = Instant.now().truncatedTo(ChronoUnit.MILLIS);
```

Comment:
What is the motivation for this change? Is it a java-level specific issue, or more of a tidying/improvement change?

Reply:
Between Java 8 and Java 9 Instant precision changed from millis to nanos. So on JDK9+ we get greater prescision. But when you serialise and deserialize Instants using Gson you still get millis precision.

In order to this test to work on all JDK versions, I eagerly reset precision to millis to unify all JDK builds.